fusor-ember-cli/app/routes/application.js

```diff
@@ -6,6 +6,7 @@ export default Ember.Route.extend({
   setupController: function(controller, model) {
     controller.set('model', model);
     var deploymentNames = [];
+    controller.set('deploymentNames', []);
```

Comment:
this fix is unrelated but I was getting some js console errors on refresh and this fix it.